admin/tests/acceptance/authenticated/users/get_test.js

```diff
@@ -165,9 +165,8 @@ module('Acceptance | authenticated/users/get', function (hooks) {
 
       // when & then #1
       await click(screen.getByRole('button', { name: 'Confirmer' }));
-      assert.dom(screen.getByText(`Prénom : (anonymised)`)).exists();
-      assert.dom(screen.getByText(`Nom : (anonymised)`)).exists();
-      assert.dom(screen.getByText(`Adresse e-mail : email_${userToAnonymise.id}@example.net`)).exists();
```

Comment:
On gagnerait à mettre à jour cette assertion supprimée avec l'assertion suivante, la regexp présentée est à la fois suffisante souple et restrictive : 

```suggestion
      assert.dom(screen.queryByText(new RegExp('Adresse e-mail : \\S+@\\S+'))).doesNotExist();
```

Reply:
Les assertions de non existence ne sont pas très robustes et la suppression en base est déjà couverte par le repo et use-case. Donc je ne pense pas qu'il soit nécessaire de le tester également dans le test d'acceptance.